### PR TITLE
Groot update DM path

### PR DESCRIPTION
**Pull request by hugolefeuvre**
FOR CONTRIBUTOR:
* [x] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] License permits unrestricted use (educational + commercial)
* [ ] This PR adds a new tool or tool collection
* [x] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)
DM path was not the same in command wrapper and json file

### CODE REVIEW

**Comment by bernt-matthias** [comment]
What's wrong with the old path?

File: data_managers/data_manager_groot_database_downloader/test-data/groot_database.loc [hunk 1]
```diff
@@ -1,2 +1,3 @@
 arg-annot.90-16102024	arg-annot (90 percent identity)	/tmp/tmpjmmj2x3o/galaxy-dev/tool-data/groot_database/data/arg-annot.90-16102024	1.1.2
 arg-annot.90-1.1.2-16102024	arg-annot (90 percent identity)	/tmp/tmpqdkgcsoq/galaxy-dev/tool-data/groot_database/data/arg-annot.90-1.1.2-16102024	1.1.2
+arg-annot.90-1.1.2-15112024	arg-annot (90 percent identity)	/tmp/tmpnlgwr_46/galaxy-dev/tool-data/arg-annot.90	1.1.2
```
Comment (bernt-matthias) [inline data_managers/data_manager_groot_database_downloader/test-data/groot_database.loc:3]: From your failing job it seems that the tool using the data tries to access `tool-data/groot_database/data/arg-annot.90-1.1.2-13112024` which is not what the new version creates. 

I definitely miss the `groot_database/` part of the tool.

**Review by bernt-matthias — COMMENTED** [review]
I guess in the config file you need to replace 

```diff
- "path": "/$database.$identity",
+ "path": "$database.$identity",
```

**Comment by hugolefeuvre** [comment]
> Likely the problem is that the data manager tool needs to place the files in a directory `$out_file.extra_file_path` instead of the job working directories.
> 
> I made the same error in a few data managers. See the fix here [Helmholtz-UFZ/galaxy-tools#71](https://github.com/Helmholtz-UFZ/galaxy-tools/pull/71)

I made the change in the last commit, hope I've done it right

File: data_managers/data_manager_groot_database_downloader/data_manager_conf.xml [hunk 2]
```diff
@@ -8,9 +8,9 @@
                 <column name="path" output_ref="out_file">
                     <move type="directory">
                         <source>${path}</source>
-                        <target base="${GALAXY_DATA_MANAGER_DATA_PATH}">groot_database/data/${value}</target>
+                        <target base="${GALAXY_DATA_MANAGER_DATA_PATH}">${path}</target>
                     </move>
-                    <value_translation>${GALAXY_DATA_MANAGER_DATA_PATH}/groot_database/data/${value}</value_translation>
+                    <value_translation>${GALAXY_DATA_MANAGER_DATA_PATH}${path}</value_translation>
```
Comment (bernt-matthias) [inline data_managers/data_manager_groot_database_downloader/data_manager_conf.xml:13]:
```
                        <target base="${GALAXY_DATA_MANAGER_DATA_PATH}">/groot_database/${path}</target>
                    </move>
                    <value_translation>${GALAXY_DATA_MANAGER_DATA_PATH}/groot_database/${path}</value_translation>
```

We should definitely put groot DBs in an extra folder.

**Review by bernt-matthias — APPROVED** [review]
Sorry. Must have overlooked this :)

**Comment by hugolefeuvre** [comment]
> Sorry. Must have overlooked this :)

No problem, thanks for the review !

**Pull request merged**